packages/pilot/src/pages/ErrorBoundary/epic.js

```diff
@@ -0,0 +1,13 @@
+import { map } from 'rxjs/operators'
+import { combineEpics, ofType } from 'redux-observable'
+import { LOGOUT_RECEIVE } from '../Account/actions'
+import { clearAllErrors } from '.'
+
+const logoutClearEpic = action$ =>
+  action$
+    .pipe(
+      ofType(LOGOUT_RECEIVE),
+      map(clearAllErrors)
+    )
+
+export default combineEpics(logoutClearEpic)
```

Comment:
aqui como tem apenas um epic não precisa do combine
```suggestion
export default logoutClearEpic
```

Reply:
Boa, feito!